features/security/src/main/feature/feature.xml

```diff
@@ -69,6 +69,12 @@
         <bundle>mvn:ddf.platform/platform-filter-delegate/${project.version}</bundle>
     </feature>
 
+    <feature name="platform-error-page-injector" version="${project.version}"
+             description="Adding a better error page to Jetty">
+        <bundle>mvn:ddf.platform.error/platform-error-api/${project.version}</bundle>
```

Comment:
❓ Why do we need to start this bundle?

Reply:
When OSGI runs the error injector looks for a error handler class that is inside the platform error api. So I want that bundle to start first

Reply:
I don't see where that is happening. To me it looks like we are using nothing in the platform-error-api bundle.